client/e2e/app.e2e-spec.ts

```diff
@@ -1,14 +1,14 @@
 import { AppPage } from './app.po';
```

Comment:
Let's delete this file, it doesn't pass test anyway 😃

Reply:
Right